docs/docsite/rst/developing_modules_general_aci.rst

```diff
@@ -0,0 +1,1851 @@
+          tenant: ansible_tenant
+          l3out: ansible_l3out
+          logical_node: lNode
+          node_id: 101
+          pod_id: 1 
+          prefix: 10.1.0.1/24
+          state: present
+        register: static1
+
+      - name: Add static routes again
+        aci_l3out_static_routes:
+          <<: *aci_info
+          tenant: ansible_tenant
+          l3out: ansible_l3out
+          logical_node: lNode
+          node_id: 101
+          pod_id: 1 
+          prefix: 10.1.0.1/24
+          state: present
+        register: static2
+        
+      - name: Add static routes containing bfd
+        aci_l3out_static_routes:
+          <<: *aci_info
+          tenant: ansible_tenant
+          l3out: ansible_l3out
+          logical_node: lNode
+          bfd: bfd
+          node_id: 101
+          pod_id: 1 
+          prefix: 10.1.0.1/24
+          state: present
+        register: static_bfd
+
+      - name: Query static routes containing bfd
+        aci_l3out_static_routes:
+          <<: *aci_info
+          tenant: ansible_tenant
+          l3out: ansible_l3out
+          logical_node: lNode
+          node_id: 101
+          pod_id: 1
+          bfd: bfd
+          prefix: 10.1.0.1/24
+          state: query
+        register: query_static_bfd
+        
+      - name: Add another static route
+        aci_l3out_static_routes:
+          <<: *aci_info
+          tenant: ansible_tenant
+          l3out: ansible_l3out
+          logical_node: lNode
+          node_id: 101
+          pod_id: 1 
+          prefix: 10.1.0.0/24
+          state: present
+        register: static_another
+
+      - name: Query all static routes
+        aci_l3out_static_routes:
+          <<: *aci_info
+          tenant: ansible_tenant
+          state: query
+        register: static_all
+
+      - name: Remove static routes
+        aci_l3out_static_routes:
+          <<: *aci_info
+          tenant: ansible_tenant
+          l3out: ansible_l3out
+          logical_node: lNode
+          node_id: 101
+          pod_id: 1
+          prefix: 10.1.0.1/24
+          state: absent
+        register: delete_static
+
+
+.. code-block:: text
+
+After inclusion of all the tasks, the configuration has been posted, modified, and deleted on our APIC. By using the values registered with results after each task, we can verify these results by comparing them with the expected response from the APIC. The result stored in the registered value is a list of dictionaries, and we access the attributes using the dot operator. If all the verifications below pass, our testing is complete.
+
+.. code-block:: yaml
+
+      - name: Verify nm_add_node
+        assert:
+          that:
+            - static1 is changed
+            - static2 is not changed
+            - static_bfd is changed
+            - static1.current.0.ipRouteP.attributes.dn == "uni/tn-ansible_tenant/out-ansible_l3out/lnodep-lNode/rsnodeL3OutAtt-[topology/pod-1/node-101]/rt-[10.1.0.1/24]"
```

Comment:
do we want to add the the children order issues between version and lookups that can be down with regex, see one of the later PRs of dev